### PR TITLE
Manylinux dockerfile 

### PR DESCRIPTION
**Pull request by micmelesse**
Creates a Dockerfile for building manylinux tensorflow wheels.

### CODE REVIEW

File: tensorflow/tools/tf_sig_build_dockerfiles/Dockerfile.rocm [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+FROM quay.io/pypa/manylinux2014_x86_64 as builder
+################################################################################
+
+# Install devtoolset build dependencies
+COPY setup.packages.rocm.sh setup.packages.rocm.sh
+COPY builder.packages.rocm.txt builder.packages.rocm.txt
+RUN /setup.packages.rocm.sh /builder.packages.rocm.txt
+
+# Install devtoolset-7 in /dt7 with glibc 2.12 and libstdc++ 4.4, for building
+# manylinux2010-compatible packages. Scripts expect to be in the root directory.
```
Comment (jayfurmanek) [inline tensorflow/tools/tf_sig_build_dockerfiles/Dockerfile.rocm]: Let's remove all of these commented out lines

File: tensorflow/tools/tf_sig_build_dockerfiles/Dockerfile.rocm [hunk 2]
```diff
@@ -0,0 +1,79 @@
+################################################################################
+FROM quay.io/pypa/manylinux2014_x86_64 as builder
+################################################################################
+
+# Install devtoolset build dependencies
```
Comment (jayfurmanek) [inline tensorflow/tools/tf_sig_build_dockerfiles/Dockerfile.rocm]: Fix the comment here. Should be something like
"# Install Dependencies"

Comment (micmelesse) [reply tensorflow/tools/tf_sig_build_dockerfiles/Dockerfile.rocm]: Done

File: tensorflow/tools/tf_sig_build_dockerfiles/Dockerfile.rocm (hunk 1 above)
Comment (micmelesse) [reply tensorflow/tools/tf_sig_build_dockerfiles/Dockerfile.rocm]: done

**Comment by jayfurmanek** [comment]
retest Ubuntu-CPU please

**Review by jayfurmanek — APPROVED** [review]

**Pull request merged**